lib/cfme/cloud_services/data_collector.rb

```diff
@@ -41,22 +39,55 @@ def raw_manifest
 
     {
       "cfme_version" => cfme_version,
-      "manifest"     => {
-        "openstack" => common.clone,
-        "rhevm"     => common.clone,
-        "vmwarews"  => common.clone,
+      "manifest" => {
+        "core" => {
+          "Zone" => {
+            "name" => nil
+          }
+        },
+        "by_provider_type" => {
+          "ManageIQ::Providers::OpenStack::CloudManager" => common.clone,
```

Comment:
Instead of using the emstype, why not just use the model name.  This also makes the fetching code consistent with the core models above.

Reply:
:+1: